### ISSUE TITLE
[Bug]: Error writing 44 bins on latest version 

### ISSUE THREAD

**Issue by Sora1995** [HiddenRamblings/TagMo#598]
```
TagMo 3.2.9 (Google Play Release) #96a1e428
Android TIRAMISU (13) - 5.33 GB RAM

--------- beginning of crash
10-22 14:45:31.886 23811 23811 E AndroidRuntime: FATAL EXCEPTION: main
10-22 14:45:31.886 23811 23811 E AndroidRuntime: Process: com.hiddenramblings.tagmo.eightbit, PID: 23811
10-22 14:45:31.886 23811 23811 E AndroidRuntime: java.lang.IllegalStateException: Fragment SettingsFragment{5d214fe} (4c2acc3f-e72a-47fb-ae3c-1a061efb6178) not attached to an activity.
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at androidx.fragment.app.Fragment.requireActivity(Fragment.java:995)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at com.hiddenramblings.tagmo.settings.SettingsFragment.buildSnackbar(SettingsFragment.java:624)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at com.hiddenramblings.tagmo.settings.SettingsFragment.lambda$parseCommitDate$35(SettingsFragment.java:642)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at com.hiddenramblings.tagmo.settings.SettingsFragment.$r8$lambda$2GSRyi1JUqwrtXcQaATbTveg1ZU(Unknown Source:0)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at com.hiddenramblings.tagmo.settings.SettingsFragment$$ExternalSyntheticLambda34.run(Unknown Source:4)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:942)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:201)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:288)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7872)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
10-22 14:45:31.886 23811 23811 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
--------- beginning of main
10-23 03:17:32.780  5971  6385 E ViewRootImpl: Accessibility content change on non-UI thread. Future Android versions will throw an exception.
10-23 03:17:32.780  5971  6385 E ViewRootImpl: android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewRootImpl$SendWindowContentChangedAccessibilityEvent.runOrPost(ViewRootImpl.java:10646)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewRootImpl.postSendWindowContentChangedCallback(ViewRootImpl.java:9486)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewRootImpl.notifySubtreeAccessibilityStateChanged(ViewRootImpl.java:9666)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.view.View.notifyViewAccessibilityStateChangedIfNeeded(View.java:14367)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.widget.TextView.setText(TextView.java:6640)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.widget.TextView.setText(TextView.java:6454)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at android.widget.TextView.setText(TextView.java:6406)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity.onTagDiscovered(NfcActivity.java:401)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity.lambda$onNewIntent$0(NfcActivity.java:249)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity.$r8$lambda$vP5NXNBqg2kDV8eg_wytd2g-zlQ(Unknown Source:0)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity$$ExternalSyntheticLambda0.run(Unknown Source:4)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:17:32.780  5971  6385 E ViewRootImpl: 	at java.lang.Thread.run(Thread.java:1012)
10-23 03:17:32.801  5971  6385 W NullPointerException: java.lang.NullPointerException
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at java.io.FileInputStream.<init>(FileInputStream.java:152)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at com.hiddenramblings.tagmo.nfctech.TagReader.readTagFile(TagReader.java:52)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at com.hiddenramblings.tagmo.nfctech.TagArray.getValidatedFile(TagArray.java:274)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity.onTagDiscovered(NfcActivity.java:405)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity.lambda$onNewIntent$0(NfcActivity.java:249)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity.$r8$lambda$vP5NXNBqg2kDV8eg_wytd2g-zlQ(Unknown Source:0)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity$$ExternalSyntheticLambda0.run(Unknown Source:4)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:17:32.801  5971  6385 W NullPointerException: 	at java.lang.Thread.run(Thread.java:1012)

10-23 03:18:02.662  6648  6648 E .tagmo.eightbit: Not starting debugger since process cannot load the jdwp agent.
10-23 03:18:02.695  6648  6648 W ziparchive: Unable to open '/data/app/~~JT2zt94u2YseJUYwrmtHfg==/com.hiddenramblings.tagmo.eightbit-LBzE8K6ZTimsXulUM-1f8Q==/split_config.arm64_v8a.dm': No such file or directory
10-23 03:18:02.695  6648  6648 W ziparchive: Unable to open '/data/app/~~JT2zt94u2YseJUYwrmtHfg==/com.hiddenramblings.tagmo.eightbit-LBzE8K6ZTimsXulUM-1f8Q==/split_config.arm64_v8a.dm': No such file or directory
10-23 03:18:02.696  6648  6648 W .tagmo.eightbit: Entry not found
10-23 03:18:02.696  6648  6648 W ziparchive: Unable to open '/data/app/~~JT2zt94u2YseJUYwrmtHfg==/com.hiddenramblings.tagmo.eightbit-LBzE8K6ZTimsXulUM-1f8Q==/split_config.en.dm': No such file or directory
10-23 03:18:02.696  6648  6648 W ziparchive: Unable to open '/data/app/~~JT2zt94u2YseJUYwrmtHfg==/com.hiddenramblings.tagmo.eightbit-LBzE8K6ZTimsXulUM-1f8Q==/split_config.en.dm': No such file or directory
10-23 03:18:02.697  6648  6648 W .tagmo.eightbit: Entry not found
10-23 03:18:02.698  6648  6648 W ziparchive: Unable to open '/data/app/~~JT2zt94u2YseJUYwrmtHfg==/com.hiddenramblings.tagmo.eightbit-LBzE8K6ZTimsXulUM-1f8Q==/split_config.xxhdpi.dm': No such file or directory
10-23 03:18:02.698  6648  6648 W ziparchive: Unable to open '/data/app/~~JT2zt94u2YseJUYwrmtHfg==/com.hiddenramblings.tagmo.eightbit-LBzE8K6ZTimsXulUM-1f8Q==/split_config.xxhdpi.dm': No such file or directory
10-23 03:18:02.698  6648  6648 W .tagmo.eightbit: Entry not found
10-23 03:18:02.702  6648  6648 W .tagmo.eightbit: JIT profile information will not be recorded: profile file does not exist.
10-23 03:18:02.702  6648  6648 W .tagmo.eightbit: JIT profile information will not be recorded: profile file does not exist.
10-23 03:18:02.702  6648  6648 W .tagmo.eightbit: JIT profile information will not be recorded: profile file does not exist.
10-23 03:18:02.715  6648  6648 W .tagmo.eightbit: Accessing hidden method Ldalvik/system/VMRuntime;->setHiddenApiExemptions([Ljava/lang/String;)V (blocked,core-platform-api, reflection, denied)
10-23 03:18:02.715  6648  6648 W BootstrapClass: reflect bootstrap failed:
10-23 03:18:02.715  6648  6648 W BootstrapClass: java.lang.reflect.InvocationTargetException
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at java.lang.reflect.Method.invoke(Native Method)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at me.weishu.reflection.BootstrapClass.<clinit>(BootstrapClass.java:29)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at me.weishu.reflection.BootstrapClass.exemptAll(Unknown Source:0)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at me.weishu.reflection.Reflection.unseal(Reflection.java:35)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at com.hiddenramblings.tagmo.TagMo.attachBaseContext(TagMo.java:58)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.Application.attach(Application.java:346)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.Instrumentation.newApplication(Instrumentation.java:1244)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1458)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1395)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6720)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2133)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.os.Handler.dispatchMessage(Handler.java:106)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.os.Looper.loopOnce(Looper.java:201)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.os.Looper.loop(Looper.java:288)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at android.app.ActivityThread.main(ActivityThread.java:7872)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at java.lang.reflect.Method.invoke(Native Method)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
10-23 03:18:02.715  6648  6648 W BootstrapClass: Caused by: java.lang.NoSuchMethodException: dalvik.system.VMRuntime.setHiddenApiExemptions [class [Ljava.lang.String;]
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at java.lang.Class.getMethod(Class.java:2103)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	at java.lang.Class.getDeclaredMethod(Class.java:2081)
10-23 03:18:02.715  6648  6648 W BootstrapClass: 	... 19 more
10-23 03:18:02.716  6648  6648 W .tagmo.eightbit: Opening an oat file without a class loader. Are you using the deprecated DexFile APIs?
10-23 03:18:02.717  6648  6648 W .tagmo.eightbit: DexFile /data/user/0/com.hiddenramblings.tagmo.eightbit/code_cache/1666513082715.dex is in boot class path but is not in a known location
10-23 03:18:02.897  6648  6681 W TagLostException: android.nfc.TagLostException: Tag was lost.
10-23 03:18:02.897  6648  6681 W TagLostException: 	at android.nfc.TransceiveResult.getResponseOrThrow(TransceiveResult.java:48)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at android.nfc.tech.BasicTagTechnology.transceive(BasicTagTechnology.java:154)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at android.nfc.tech.NfcA.transceive(NfcA.java:120)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.transceive(NTAG215.java:78)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.readSignature(NTAG215.java:182)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.TagReader.getBankSignature(TagReader.java:113)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.ScanTag.onTagDiscovered(ScanTag.java:81)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.lambda$onNewIntent$66(BrowserActivity.java:2625)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.$r8$lambda$cogEJdf83ad7mxWLbCcVMOUyc70(Unknown Source:0)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity$$ExternalSyntheticLambda18.run(Unknown Source:4)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:18:02.897  6648  6681 W TagLostException: 	at java.lang.Thread.run(Thread.java:1012)
10-23 03:18:02.964  6648  6681 W TagLostException: android.nfc.TagLostException: Tag was lost.
10-23 03:18:02.964  6648  6681 W TagLostException: 	at android.nfc.TransceiveResult.getResponseOrThrow(TransceiveResult.java:48)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at android.nfc.tech.BasicTagTechnology.transceive(BasicTagTechnology.java:154)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at android.nfc.tech.NfcA.transceive(NfcA.java:120)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.transceive(NTAG215.java:78)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.lambda$amiiboFastRead$1(NTAG215.java:229)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.$r8$lambda$vWciRVdbZ3Ld-OoUB_Jc1KdIW3g(Unknown Source:0)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215$$ExternalSyntheticLambda0.doFastRead(Unknown Source:2)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.internalFastRead(NTAG215.java:254)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.amiiboFastRead(NTAG215.java:229)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.TagReader.readBankTitle(TagReader.java:86)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.TagReader.readTagTitles(TagReader.java:94)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.ScanTag.onTagDiscovered(ScanTag.java:87)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.lambda$onNewIntent$66(BrowserActivity.java:2625)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.$r8$lambda$cogEJdf83ad7mxWLbCcVMOUyc70(Unknown Source:0)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity$$ExternalSyntheticLambda18.run(Unknown Source:4)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:18:02.964  6648  6681 W TagLostException: 	at java.lang.Thread.run(Thread.java:1012)
10-23 03:18:02.964  6648  6681 W TagReader: Failed to parse banks!
1beforehand.  Releasing leaked death recipient: com.google.android.play.core.internal.zzai
10-23 03:18:05.654  6648  6793 W TagLostException: android.nfc.TagLostException: Tag was lost.
10-23 03:18:05.654  6648  6793 W TagLostException: 	at android.nfc.TransceiveResult.getResponseOrThrow(TransceiveResult.java:48)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at android.nfc.tech.BasicTagTechnology.transceive(BasicTagTechnology.java:154)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at android.nfc.tech.NfcA.transceive(NfcA.java:120)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.transceive(NTAG215.java:78)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.readSignature(NTAG215.java:182)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.TagArray.isElite(TagArray.java:91)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.ScanTag.onTagDiscovered(ScanTag.java:65)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.lambda$onNewIntent$66(BrowserActivity.java:2625)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.$r8$lambda$cogEJdf83ad7mxWLbCcVMOUyc70(Unknown Source:0)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity$$ExternalSyntheticLambda18.run(Unknown Source:4)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:18:05.654  6648  6793 W TagLostException: 	at java.lang.Thread.run(Thread.java:1012)
10-23 03:18:07.297  6648  6793 W TagLostException: android.nfc.TagLostException: Tag was lost.
10-23 03:18:07.297  6648  6793 W TagLostException: 	at android.nfc.TransceiveResult.getResponseOrThrow(TransceiveResult.java:48)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at android.nfc.tech.BasicTagTechnology.transceive(BasicTagTechnology.java:154)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at android.nfc.tech.NfcA.transceive(NfcA.java:120)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.NTAG215.readPages(NTAG215.java:97)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.TagReader.readFromTag(TagReader.java:69)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.nfctech.ScanTag.onTagDiscovered(ScanTag.java:95)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.lambda$onNewIntent$66(BrowserActivity.java:2625)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity.$r8$lambda$cogEJdf83ad7mxWLbCcVMOUyc70(Unknown Source:0)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at com.hiddenramblings.tagmo.browser.BrowserActivity$$ExternalSyntheticLambda18.run(Unknown Source:4)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:18:07.297  6648  6793 W TagLostException: 	at java.lang.Thread.run(Thread.java:1012)

10-23 03:21:09.105  6648  7679 E ViewRootImpl: Accessibility content change on non-UI thread. Future Android versions will throw an exception.
10-23 03:21:09.105  6648  7679 E ViewRootImpl: android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewRootImpl$SendWindowContentChangedAccessibilityEvent.runOrPost(ViewRootImpl.java:10646)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewRootImpl.postSendWindowContentChangedCallback(ViewRootImpl.java:9486)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewRootImpl.notifySubtreeAccessibilityStateChanged(ViewRootImpl.java:9666)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3848)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.view.View.notifyViewAccessibilityStateChangedIfNeeded(View.java:14367)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.widget.TextView.setText(TextView.java:6640)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.widget.TextView.setText(TextView.java:6454)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at android.widget.TextView.setText(TextView.java:6406)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity.onTagDiscovered(NfcActivity.java:401)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity.lambda$onNewIntent$0(NfcActivity.java:249)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity.$r8$lambda$vP5NXNBqg2kDV8eg_wytd2g-zlQ(Unknown Source:0)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at com.hiddenramblings.tagmo.NfcActivity$$ExternalSyntheticLambda0.run(Unknown Source:4)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:21:09.105  6648  7679 E ViewRootImpl: 	at java.lang.Thread.run(Thread.java:1012)
10-23 03:21:09.126  6648  7679 W NullPointerException: java.lang.NullPointerException
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at java.io.FileInputStream.<init>(FileInputStream.java:152)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at com.hiddenramblings.tagmo.nfctech.TagReader.readTagFile(TagReader.java:52)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at com.hiddenramblings.tagmo.nfctech.TagArray.getValidatedFile(TagArray.java:274)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity.onTagDiscovered(NfcActivity.java:405)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity.lambda$onNewIntent$0(NfcActivity.java:249)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity.$r8$lambda$vP5NXNBqg2kDV8eg_wytd2g-zlQ(Unknown Source:0)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at com.hiddenramblings.tagmo.NfcActivity$$ExternalSyntheticLambda0.run(Unknown Source:4)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
10-23 03:21:09.126  6648  7679 W NullPointerException: 	at java.lang.Thread.run(Thread.java:1012)
```

Edit: Removed all the expected failures and debugging info

**Comment by AbandonedCart** [comment]
Next time, just use the form. It does all the formatting for you.

**Comment by AbandonedCart** [comment]
```
10-23 03:21:09.105  6648  7679 E ViewRootImpl: Accessibility content change on non-UI thread. Future Android versions will throw an exception.
10-23 03:21:09.105  6648  7679 E ViewRootImpl: android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
```

Fixed by 93ccf8c975a77981b2e6a04e881bb8bd68e518d8

**Comment by AbandonedCart** [comment]
```
10-23 03:21:09.126  6648  7679 W NullPointerException: java.lang.NullPointerException
```

Fixed by 662c8a7445a2cc24ed3d9def27062d8d17587b66

**Comment by AbandonedCart** [comment]
```
10-23 03:18:02.715  6648  6648 W BootstrapClass: reflect bootstrap failed:
10-23 03:18:02.715  6648  6648 W BootstrapClass: java.lang.reflect.InvocationTargetException
```

Fixed by d05fe1d4280a13effd67deeb8dd01cb2fcd3f1ae

**Comment by AbandonedCart** [comment]
```
10-23 03:18:05.654  6648  6793 W TagLostException: android.nfc.TagLostException: Tag was lost.
```

That one is removing the tag from the NFC field, so there is nothing to address.
